src/app/core/cases/cases.service.spec.ts

```diff
@@ -425,4 +429,68 @@ describe('CasesService', () => {
         });
     });
   });
+
+  describe('getCaseHistoryView()', () => {
+    const EVENT: CaseViewEvent = {
```

Comment:
recently we moved away from direct instantiation of objects in tests with the object literal notation because makes the test very fragile and reduces maintainability. 
We try to define fixture files with creational methods that can be reused by many tests. Can you have a look at:
`profile.test.fixture.ts`

Reply:
Done for case history